### DISCUSSION TITLE
Nixie and REM CHEM Chemistry Control Naive Questions

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#915] [REM] [answered]
I'm finally migrating my pool to full Nixie & REM CHEM chemistry control and have some stupid newbie questions.

I see where I can set the volume of my acid tank, but I have not found where to set the level of the acid tank.  So, for my switchover to REM CHEM, I am getting the alarm "pH Tank Empty."  How do I clear the alarm and provide REM with the current volume of acid in the tank?

I don't want to obsess about the CSI in my pool, but I get very different values for CSI depending on the calculator I use, including REM's.  First, here are the current measurements for my pool--using Taylor and probe values:

20,000 gallons
plaster pool (Pebble Tech)
Water temp 67.3 Fahrenheit aka 19.65 Centigrade
Cl  1.8 ppm  (taylor)
pH  7.6 (Taylor) 7.2 Atlas probe, calibrated last week
Alk  90  (Taylor)
Ca  320  (Taylor)
Cyn 25-30  (Taylor) always challenging to decipher, but more or less what I strive for, the pool has a pool screen
Salt 2900 Intellichlor--a bit low, will add another 40 lbs

REM CSI  -0.58
REM LSI -.0205
Using the Taylor Watergram slide rule with a pH of 7.2, I get a CSI of about -0.3
Using the Taylor Watergram slide rule with a pH of 7.6, I get a CSI of about 0.1

Other than the salt concentration being a bit low, which shouldn't affect CSI, is the pool in a safe range for CSI?  REM is complaining, of course, saying corrosion may occur when I use CSI, but it's happy when I use LSI.  Overall, I thought the chemistry ins't too bad.  I also assume the "real" pH is somewhere between the probe value & the Taylor value & reasonable either way.

REM pH target is 7.2 and ORP target is 725 mv

Thanks for your help!

**Comment by rstrouse** [comment, marked as answer]
On the dashBoard click rem chem then the tank.

**Reply by FlaMike** [reply]
Thanks!  That was easy enough.  Had no idea it was hiding in plain sight!!!!
